input/pagecontent/index.md

```diff
@@ -117,18 +117,20 @@ This guide is divided into several pages which are listed at the top of each pag
   - [Medicine Information](medicine-information.html): This page provides guidance on constructing medications related resources.
   - [AUCDI](aucdi.html): This page maps the AU Core resources and elements to AUCDI data classes and data elements.
   - [Relationship with other IGs](relationship.html): This page provides guidance on the relationship between AU Core and other implementation guides.
-  - [Comparison with other national and international specifications](comparison.html): This page provides comparison between AU Core profiles and other national, or international implementation guides.
   - [AU Variance Statement](variance.html): This page documents the variance from AU Base.
   - [Comparison with other national and international IGs](comparison.html): This page provides comparison between AU Core profiles and other national, or international implementation guides.
   - [Future of AU Core](future.html): This page outlines the approach to developing AU Core and yearly update cycle.
+- [Security and Privacy](security.html): This page outlines the security and privacy obligations to consider when implementing AU Core.
 - [FHIR Artefacts](artifacts.html): These pages provide detailed descriptions and formal definitions for all the FHIR artefacts defined in this guide.
   - [Profiles and Extensions](profiles-and-extensions.html): This set of pages describes the profiles and extensions that are defined in this guide to exchange quality data. Each profile page includes a narrative description and guidance, formal definition and a "Quick Start" guide which summarises the supported search transactions for each profile. Although the guidance typically focuses on the profiled elements, it may also may focus on un-profiled elements to aid with implementation.
   - [Search Parameters](search-parameters.html): This set of pages lists the search parameters extended for use in this guide for use in AU Core operations.
   - [Terminology](terminology.html): This set of pages lists the value sets and code systems defined in this guide.
   - [Capability Statements](capability-statements.html): This set of pages define the expected FHIR capabilities of AU Core Responders and Requesters.
   - [Actor Definitions](actors.html): This set of pages define the AU Core actors, AU Core Responder and AU Core Requester.
 - [Examples](examples.html): This page lists all the examples used in this guide.
-- [Downloads](downloads.html): This page provides links to downloadable artefacts that developers can use to help them implement this guide.
+- [Support](support.html): These pages provide supporting material for implementation or AU Core.    
+  - [Downloads](downloads.html): This page provides links to downloadable artefacts that developers can use to help them implement this guide.
```

Comment:
This text describing the downloads page is different across all three specs. There is an open PR on AU Base that is not consistent with AU Core.

Advise on the text that will be applied and make both PRs consistent; also create the AU eRequesting PR and submit. Approval of all three spec PRs will occur in parallel.

Reply:
Changed all to: "This page provides links to downloadable artefacts."